<launch>

	<node pkg="vizon" type="vizon" name="vicon" respawn="true">
		<param name="vicon_IP" value="192.168.1.50:801" />
	</node>
	
	<node pkg="connectivity_controller_old" type="connectivity_old" name="connectivity" respawn="true">
		<param name="robot1name" value="gandhi" />
		<param name="robot2name" value="suleyman" />
		<param name="robot3name" value="alexander" />
		<param name="robot4name" value="hiawatha" />
		<param name="robot5name" value="darius" />
		<param name="robot6name" value="caesar" />
 	</node>
 	
 	<node pkg="rqt_reconfigure" type="rqt_reconfigure" name="reconfigure_gui"/>
	
	
</launch>
